.env.example

```diff
@@ -0,0 +1,2 @@
+# VCR Application ID for deploying to VCR from local machine
+VCR_APP_ID=
```

Comment:
Question : Why are we calling this VCR_APP_ID rather than VONAGE_APP_ID?

Reply:
VONAGE_APP_ID is already in the backend .env for using vonage unified video environment. I think it would be confusing, this specifies that this is for use by the VCR script.